assets/style.css

```diff
@@ -253,6 +262,11 @@ body.night {
   background-color: var(--dark-purple);
 }
 
+.night .resources:hover {
+  color: #ff0037;
```

Comment:
it makes sense to use css variable for it

Reply:
What happened can u please describe it?
Because I am a beginner so it will be helpful for me I am gonna learn new things .

Reply:
I am a beginner so can you please tell me why?

Reply:
> it makes sense to use css variable for it
- Are you asking or telling?

Reply:
I am not the maintainer of this project, but it looks like all CSS file uses CSS variable, so it makes sense to follow the general style + For some colors, you used twice, it is better to not repeat it and create a variable for it.

Reply:
What my colleague is trying to tell you here is that if we use the same color in more than one place in the CSS file we usually will save it to a variable. If you look right at the top of the CSS file you will see a block called `:root` and within it you see colors that are given names like `--light-grey`. These are the variables. And you can then use them in any part of the CSS file with the syntax `var(--light-grey)`. So you can add the color to the variables on top and give it a name, then use this var syntax instead of the color.

Reply:
> I am not the maintainer of this project, but it looks like all CSS file uses CSS variable, so it makes sense to follow the general style + For some colors, you used twice, it is better to not repeat it and create a variable for it.

Thanks for sharing your experience 🙌